Editor/BugSplat.Unity.Editor.asmdef

```diff
@@ -1,7 +1,9 @@
 {
     "name": "BugSplat.Unity.Editor",
     "rootNamespace": "",
-    "references": [],
+    "references": [
+        "GUID:a8de7f358e223354dbef7a09cad00841"
```

Comment:
Where did this come from?

Reply:
Editor needs a reference to Runtime in order for BugSplatOptionsEditor to know about BugSplatOptions.